tests/cucumber/e2e/pageObjects/SearchPage.js

```diff
@@ -0,0 +1,31 @@
+// import * as dotenv from "dotenv";
+class SearchPage {
+    constructor() {
+        this.baseUrl = "http://localhost:8000/";
+        this.searchBtn = `//button[@class="btn btn-outline-light m-1"]`;
+        this.errorMsg = `//div[@class="alert alert-danger"]`;
+        this.searchRes = `//a[@class="`;
+        this.searchInp = `#searchTextBox`;
+    }
+
+    async searchAction(movieName) {
+        await this.search(movieName);
+    }
+
+    async homePage() {
+        await page.goto(this.baseUrl);
+    }
+
+    async search(movieName) {
+        await page.fill(this.searchInp, movieName);
+        await page.click(this.searchBtn);
+    }
+
+    async checkSearchResult(movieName) {
```

Comment:
This function does not check anything but only returns the search text that is how a function in the pageobject should operate. :+1: 

The only small improvement to make is the naming, it is good to have a name, so that the reader of the code can know what the function does, just by seeing the name.
```suggestion
    async getSearchResult(movieName) {
```